index.js

```diff
@@ -20,10 +20,33 @@ const { parseFixed } = require("@ethersproject/bignumber");
 // --proposerReward: Proposal reward to be forwarded to the created contract to be used to incentivize price proposals.
```

Comment:
Can you add instructions to the readme for different params that could be used?

Reply:
done